modules/creating-kubefed-CRs.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * kubefed/installing-kubefed.adoc
+
+[id="creating-kubefed-CRs_{context}"]
+= Creating KubeFed custom resources
+
+You must create the `KubeFedWebHook` and `KubeFed` custom resources (CRs) for {KubeFedProductShortName} deployment.
+The {KubeFedProductName} operator will create the required custom resource definitions (CRDs) for these CRs automatically.
+
+.Prerequisites
+* You must have the `oc` CLI tool installed.
+
+.Procedure
+
+. Create a `KubeFedWebHook` resource to instantiate an admission webhook controller for {KubeFedProductShortName}.
++
+----
+$ cat <<-EOF | oc apply -n <namespace> -f -
+ ---
+ apiVersion: operator.kubefed.io/v1alpha1
+ kind: KubeFedWebHook
+ metadata:
+   name: kubefed-webhook-resource
+ spec:
```

Comment:
I would make a small change here: `spec: {}`. 
CRD validation doesn't accept empty values for type "object" fields
https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_kubefed-operator/64/pull-ci-openshift-kubefed-operator-master-olm-e2e/149
@pmorie wdyt

Reply:
@pmorie can you confirm please

Reply:
@abrennan89 No action required from your side. It should be `spec:` the way it's right now.

Reply:
Thanks for confirming, Sohan!